Resources/public/css/sb-admin-2.css

```diff
@@ -0,0 +1,354 @@
+/*!
```

Comment:
TODO, remove these assets from the PHP world and let dedicated tools handle these.